### PR TITLE
added new version

### PR DESCRIPTION
**Pull request by ArbelNathan**
added a new version to the IBM-block-storage project
```release-note
NONE
```

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [HyperV CSI](https://github.com/Zetanova/hyperv-csi-driver) | `eu.zetanova.csi.hyperv` | v1.0, v1.1 | A Container Storage Interface (CSI) driver to manage hyperv hosts | Persistent | Read/Write Multiple Pods | Yes | 
-[IBM Block Storage](https://github.com/ibm/ibm-block-csi-driver) | `block.csi.ibm.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) [Driver](https://www.ibm.com/support/knowledgecenter/SSRQ8T) for IBM Spectrum Virtualize, IBM Storwize, IBM FlashSystem 9100, A9000 and A9000R, IBM DS8880 and DS8900. | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
+[IBM Block Storage](https://github.com/ibm/ibm-block-csi-driver) | `block.csi.ibm.com` | v1.0, v1.1, v1.2, v1.3 | A Container Storage Interface (CSI) [Driver](https://www.ibm.com/support/knowledgecenter/SSRQ8T) for IBM Spectrum Virtualize, IBM Storwize, IBM FlashSystem 9100, A9000 and A9000R, IBM DS8880 and DS8900. | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
```
Comment (xing-yang) [inline book/src/drivers.md:55]: Column 3 is for "Compatible with CSI Version(s)".  I wonder if you meant to add a new IBM CSI driver version?

Comment (ArbelNathan) [reply book/src/drivers.md:55]: oh, yes

Comment (xing-yang) [reply book/src/drivers.md:55]: We actually don't have a column for adding the CSI driver version here.

It looks that the compatible CSI version for this driver should be v1.0, v1.1?
https://github.com/IBM/ibm-block-csi-driver/blob/develop/go.mod#L6

Comment (ArbelNathan) [reply book/src/drivers.md]: you are right we will progress in csi version next publication. fixed. done

**Comment by ArbelNathan** [comment]
plz close

**Pull request closed without merging**